src/test/java/reposense/parser/ArgsParserTest.java

```diff
@@ -699,10 +698,12 @@ public void parse_withDatesAndTimezone_success() throws Exception {
                 .addUntilDate("30/11/2017")
                 .build();
         CliArguments cliArguments = ArgsParser.parse(translateCommandline(input));
-        int[] expectedSinceTime = {21, 0, 0};
-        Date expectedSinceDate = TestUtil.getDate(2017, Calendar.JUNE, 30, expectedSinceTime);
-        int[] expectedUntilTime = {20, 59, 59};
-        Date expectedUntilDate = TestUtil.getDate(2017, Calendar.NOVEMBER, 30, expectedUntilTime);
+        int[] expectedSinceTime = {0, 0, 0};
+        LocalDateTime expectedSinceDate = TestUtil.getDate(2017,
+                Month.JULY.getValue(), 1, expectedSinceTime);
+        int[] expectedUntilTime = {23, 59, 59};
+        LocalDateTime expectedUntilDate = TestUtil.getDate(2017,
+                Month.NOVEMBER.getValue(), 30, expectedUntilTime);
```

Comment:
This changes the original intent of the test. Is it possible to test such that we know the `--timezone` argument is correctly parsed?

Reply:
For this one, I'm not sure about the UTC times. Example for `expectedSinceTime = {21, 0, 0}`: 

Since the time for a `since date` is set to midnight for the timezone which it is in (in this case UTC+11), the `expectedSinceTime` should be `13:00:00` in UTC.

Reply:
![Screenshot (67)](https://user-images.githubusercontent.com/69678785/152346411-ab36d1e1-4427-4db0-9563-08a53e8df673.png)

In the original implementation of `getZonedSinceDate`, it appears that the midnight time was subtracted by 3 hours (`systemRawOffset - zoneRawOffset`). Likewise for `getZonedUntilDate`.

Reply:
Yup, the subtraction was to account for the time zone, as previously the since and until dates would be saved in the given time zone. However, as now the since and until dates are stored as `LocalDateTime` with no zone information, I was wondering if we can test if the `--timezone` flag is correctly used, which was the original intent of the test.

It does not have to be right here in this test function, especially since the implementation has changed quite a bit, but we still need to test to see if the `--timezone` flag is still working.

Reply:
I see. It makes sense if the original expected times were written in terms of UTC+8. 

Because I'm currently writing the expected times in terms of UTC to follow Github Actions.